methods.py

```diff
@@ -4,7 +4,6 @@
 import pickle

```

Comment:
I focus a bit on naming your functions/variables. It might seem like a bit overkill, since it might seem clear for this number of functions. However from experience I know that if you look back at your own code a (few) month(s) later, you are already quickly confused, if the naming is not really clear. Making it really worth spending a bit extra time on naming functions and variables, so the code becomes easy to understand.